File: dayFour/afterBreak/python/taskFour.py
sum_total = 0
even_index_counter = 0
even_index_counter = int(even_index_counter)

print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    if scores % 2 == 0:
        sum_total = sum_total + score
        even_index_counter += 1
print("The sum of even-indexed scores is:", sum_total)

File: dayFour/afterBreak/java/TaskFour.java
import java.util.*;


public class TaskFour {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int evenIndexSum = 0;
	
	System.out.println("Enter the scores below: ");
	int counter= 0;
	
	for(; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    if(counter != 0 && counter % 2 == 0)
	      {
	        evenIndexSum += score;
	      } else { }
	  }
	  
	System.out.println("The sum of even-index scores is: " + evenIndexSum);
	}
}
